File: scripts/cats_bats_spiders.php
<?php
    /**
     * Author: Damian Dabrowski
     * Language: PHP
     * Github: https://github.com/ddcodepl
     */

    $elements = [];
    for ($i = 1; $i <= 100; $i++) {
        if ($i % 3 === 0) {
            if ($i % 5 === 0) {
                $elements[$i] = "Spiders ";
            } else {
                $elements[$i] = "Cats ";
            }
        } else if ($i % 5 === 0) {
            $elements[$i] = "Bats ";
        } else {
            $elements[$i] = "$i ";
        }
    }

    print_r($elements);

File: scripts/CatsBatsSpiders.pas
// AUTHOR: Damian Dabrowski
// LANGUAGE: Pascal
// GITHUB: https://github.com/ddcodepl

Program CatsBatsSpiders(output);
var i : integer;

begin
    for i := 1 to 100 do
    begin
        if ((i mod 3) = 0)
        then
            begin
                if ((i mod 5) = 0)
                    then writeln('Spiders')
                    else writeln('Cats');
            end
        else if ((i mod 5) = 0)
            then writeln('Bats')
            else writeln(i);
    end;
end.
